rcjaRegistration/events/forms.py

```diff
@@ -26,3 +30,21 @@ def __init__(self, *args, user, event, **kwargs):
         self.fields['event'].initial = event.id
         self.fields['event'].disabled = True
         self.fields['event'].widget = forms.HiddenInput()
+
+def getSummaryForm(request):
+    # Use constructor function as user from request is required for permissions
+    class SummaryRequestForm(forms.Form):
+        states = []
+        for i, state in enumerate(State.objects.all()):
+            if checkCoordinatorPermission(request, State, state, 'view'):            
+                states.append((str(i),state.name))

```

Comment:
Is this supposed to bind the ID of the state to the name, rather than the index of the array?

When I render this the values aren't matched with the IDs so the page fails to load depending on which one you pick
![image](https://github.com/user-attachments/assets/1ea2efa8-53df-4740-b202-61a5410a7c56)
![image](https://github.com/user-attachments/assets/c5a4dc41-3982-4d5c-9ad4-779b1029d7b2)

If you have a look at the dropdown in admin you can find the actual state IDs
![image](https://github.com/user-attachments/assets/3131a517-ec13-491a-afb9-b334c97cfc5a)